cylc/flow/batch_sys_handlers/background.py

```diff
@@ -57,6 +84,20 @@ def submit(cls, job_file_path, submit_opts):
             # process can detach as a process group leader and not subjected to
             # SIGHUP from the current process.
             # TODO: close stdout? Maybe atexit?
+            env = {
```

Comment:
I'm not sure we need most of these. We shouldn't need anything which isn't defined when we submit a job via ssh. That's why, for example, `CYLC_VERSION` is defined in the job script.

Reply:
I'll re-think this tomorrow - in my original hack I did this for the the first subprocess (that executes `cylc jobs-submit`) and `CYLC_VERSION` is (or may be) needed by the cylc wrapper for that... then transferred my `env` dict to the background-specific subprocess.

Reply:
I've removed all the extra variables - on the first pass I was thinking (erroneously) that this also affected the shell in which `cylc jobs-submit` is executed, so `CYLC_VERSION` etc. would be needed to find the right Cylc version there (no job script to export the variables yet), but it doesn't - it's only for the batch system-specfic submit command shell.